### PR TITLE
update script

### PR DESCRIPTION
**Pull request by oshercc**

### CODE REVIEW

File: release/main.py [hunk 1]
```diff
@@ -31,6 +31,8 @@ def tag_all(manifest, tag, delete_if_exists=False):
     gtools = gittools.GitApiUtils()
     repos_with_tag = []
     for repo in manifest_contnet.keys():
+        if not repo.startswith("openshift/"):
+            continue
```
Comment (eranco74) [inline release/main.py:35]: Why skip if it's not openshift?
Add logging

File: release/gittools.py [hunk 2]
```diff
@@ -53,7 +53,7 @@ def delete_tag(self, repo, tag):
         :param str tag: The tag to delete (e.g. 'v1.0.0')
         """
         logging.info('Deleting tag %(tag)s in repository: %(repo)s', dict(tag=tag, repo=repo))
-        ref_url = "%s/%s/git/refs/tags/%s" % (self.GIT_API_REPOS, repo, tag)
+        ref_url = "%s/%s/tags/%s" % (self.GIT_API_REPOS, repo, tag)
```
Comment (eranco74) [inline release/gittools.py:56]: Sure this is the url?

Comment (oshercc) [reply release/gittools.py:56]: I was also surprised,
but this works and the old isn't working.

File: release/main.py (hunk 1 above)
Comment (oshercc) [reply release/main.py:36]: we don't have permission to tag it, adding logs

Comment (ronniel1) [reply release/main.py:36]: @oshercc  I do have permission to tag it
so we should not skip it

**Pull request closed without merging**